hydra-node/test/Hydra/OptionsSpec.hs

```diff
@@ -96,9 +98,13 @@ spec = parallel $
       setFlags ["--monitoring-port", "65535"]
         `shouldParse` Run defaultRunOptions{monitoringPort = Just 65535}
 
-    it "parses --version flag as a parse error" $
-      shouldNotParse ["--version"]
-
+    it "flag --version returns version with base version from cabal" $ do
+      case parseHydraCommandFromArgs ["--version"] of
+        Failure p ->
+          let (v, _, _) = execFailure p ""
+           in (show v :: String)
```

Comment:
Should: use [`renderFailure`](https://hackage.haskell.org/package/optparse-applicative-0.18.1.0/docs/Options-Applicative.html#v:renderFailure) instead as it seems to be the canonical function to get the output as printed originally.